yb-voyager/src/srcdb/mysql.go

```diff
@@ -164,6 +164,10 @@ func (ms *MySQL) ExportSchema(exportDir string) {
 	ora2pgExtractSchema(ms.source, exportDir)
 }
 
+func (ms *MySQL) GetIndexesInfo() *[]utils.IndexInfo {
```

Comment:
I guess it's fine to defer the implementation of other DBs for later, but let's discuss this once in the standup...

Reply:
done - https://yugabyte.atlassian.net/browse/DB-8260